internal/common/agent_util_unix.go

```diff
@@ -66,6 +73,51 @@ func TouchFile(filePathAbsolute string) error {
 	return nil
 }
 
+// printOutputAndError does nothing if there was no error.
+// Else it prints stdout and stderr.
+func printOutputAndError(stdout []byte, err error) {
+	if err == nil {
+		return
+	}
+	stderr := ""
+	ee, ok :=  err.(*exec.ExitError)
+	if ok {
+		stderr = string(ee.Stderr)
+	}
+	log.Printf("failed\n\tstdout:\n%s\n\tstderr:\n%s\n", string(stdout), stderr)
+}
+
+func UninstallAgent(pm PackageManager) error {
+	log.Printf("Uninstalling Agent...")
+	var c *exec.Cmd
+	switch pm {
+	case RPM:
+		c = exec.Command("bash", "-c", "sudo rpm -e amazon-cloudwatch-agent")
+	case DEB:
+		c = exec.Command("bash", "-c", "sudo dpkg -r amazon-cloudwatch-agent")
+	default:
+		log.Fatalf("unsupported package manager, %v", pm)
+	}
+	out, err := c.Output()
+	printOutputAndError(out, err)
+	return err
+}
+
+// InstallAgent can determine the package manager based on the installer suffix.
+func InstallAgent(installerFilePath string) error {
+	log.Printf("Installing Agent...")
+	var c *exec.Cmd
+	// Assuming lower case
```

Comment:
we could transform the path to be lowercase to enforce it, but I don't think it's super important to make that change